src/renderer/SiteInfoMenuItem.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useStoreSelector, selectors } from './store/store';
+import {
+	Spinner,
+} from '@getflywheel/local-components';
+import { OptimizerStatus } from '../types';
+
+const SiteInfoMenuItem: React.FC = () => {
+	const selectedSiteImages = useStoreSelector(selectors.selectedSiteImages);
+
+	const selectedImages = useStoreSelector(selectors.selectedSiteImages);
+	const uncompressedImages = useStoreSelector(selectors.uncompressedSiteImages);
+
+	const isSelectableTable = useStoreSelector(selectors.selectActiveSiteIsSelectableTable);
+	const isStatusRunning = useStoreSelector(selectors.selectActiveSiteIsStatusRunning);
+	const data = OptimizerStatus.BEFORE ? uncompressedImages : selectedImages;
+	const selectedCount = data.filter(data => data.isChecked).length
+
+	return (
+		<div className="SiteInfoMenuItem">
+			<span style={{ flexGrow: 1 }}>Image Optimizer</span>
+			{ isSelectableTable && selectedCount
```

Comment:
If I'm understanding this correctly, we only ever want a `Spinner` OR a `Badge`, correct?

If so, it would be nice to condense this down to a single ternary to help guard against any edge cases where neither render or both render simultaneously.

Reply:
Unfortunately, there're 3 states:
- spinner
- badge
- nothing

I'm all ears on more graceful ways of handling it, though.

Reply:
Aha! I see the issue. The only thing that comes to mind is something like (condensed sudo code)

```ts
let Icon = null;

if (isSelectableTable && selectedCount) {
    Icon = <Badge />;
}
if (isStatusRunning) {
    Icon = <Spinner />
}

render (
    <Stuff>
        <Icon />
    <Stuff />
)
```

I don't really know if that's better though, just different.

Reply:
I'm personally okay either way you take this! 🙌

Reply:
Cool, I like that better. I'll make that change. Thanks!